src/appshell/view/firstlaunchsetup/firstlaunchsetupmodel.cpp

```diff
@@ -108,7 +108,12 @@ bool FirstLaunchSetupModel::askAboutClosingEarly()
                                  buttons,
                                  int(IInteractive::Button::Cancel));
 
-    return result.standardButton() == IInteractive::Button::Cancel;
+    auto button = result.standardButton();
+    if (button == IInteractive::Button::Cancel) {
+        interactive()->open("musescore://home?section=projects");
```

Comment:
The askAboutClosingEarly method only asks and returns the user's choice.
The logic of what we should do with a certain user choice should be stored in the StartupScenario where we call FIRST_LAUNCH_SETUP dialog